app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -58,6 +58,18 @@
          <option value="blue">Blue</option>
       </select>
    </div>
+   <div class="user_categories">
+      <select name="the_categories">
```

Comment:
Add label

Reply:
Noticed the file was moved to templates/partials/calendar/event so I did what you asked me over there.
All the notes have been fixed, ready for merge :)